incubator/minecraft/templates/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rcon-password: {{ default "" .Values.MinecraftServer.Rcon.Password | b64enc | quote }}
```

Comment:
They're not too secret, but I'm not sure why I'd make it a configmap instead. It looks like Secrets are used in some of the other charts.

Hopefully the Secret situation improves as Kubernetes matures. That'd be what I'm banking on in using Secrets, as flimsy as they are now.